_clubs/naak/naak-earthcorps-projects-for-2019.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: default 
+title: NAAK / EarthCorps Projects for 2019
+date: 2019-07-14 02:25
+author: Klahanie
+comments: true
+categories: [naak]
+---
+
+# NAAK / EarthCorps Projects for 2019
+
+#### Yellow Lake Native Growth Protection Area (NGPA)
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Yellow archangel, Purple loosestrife, Tansy ragwort and Garlic mustard.
+
+#### North Fork of Issaquah Creek (NGPA)
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Yellow archangel, Purple loosestrife, Tansy ragwort and Garlic mustard.
+
+#### Reforestation of Yellow Lake NGPA with 1 gallon native tree stock
+Add native trees including western red cedar and western hemlock to specific areas for reforestation in the fall of the year.  TBD based upon need and sustainability.
+
+#### Mountain View NGPA (including Scotch broom behind pickleball court)
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Tansy ragwort and English holly.
+
+#### Klahanie Perimeter NGPA
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Tansy ragwort and Yellow Archangel.  All control work will be based upon potential impacts of the Issaquah/Fall City road expansion.
+
+#### Pea Patch Area (NGPA)
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries and Tansy ragwort.
+
+#### Sutter Pond, Gander Pond, Lake Park, Heatherwood, 242<sup>nd</sup> wetland, 242<sup>nd</sup> Park and Native Plant Demonstration Garden (multiple small natural areas)
+Continue control of noxious weeds such as English ivy, Scotch broom, Himalayan blackberries, Tansy ragwort, Japanese knotweed and Purple loosestrife.  Increase level of compost in Native Plant Demonstration Garden and add native plants, as needed.  Water plantings from 2018 in Lake Park, as needed.

```

Comment:
Add Oxford commas
"Park, and Native Plant Demonstration"
"Japanese knotweed, and Purple"